modules/rosa-creating-cluster.adoc

```diff
@@ -6,7 +6,7 @@
 [id="rosa-creating-cluster_{context}"]
 = Creating your cluster
 
-You can create an {product-title} cluster using the `rosa` CLI.
+You can create an {product-title} cluster using the {product-title} (ROSA) CLI (`rosa`).
```

Comment:
This should be "a {product-title} (ROSA) cluster" and then "using the ROSA CLI (`rosa`)" I think